src/scripts/genEventing.py

```diff
@@ -530,6 +592,53 @@ def generatePlatformIndependentFiles(sClrEtwAllMan, incDir, etmDummyFile, extern
             #vm header:
             Clrallevents.write(generateClrallEvents(eventNodes, allTemplates) + "\n")
 
+            providerName = providerNode.getAttribute('name')
+            providerSymbol = providerNode.getAttribute('symbol')
+
+            eventpipeProviderCtxName = providerSymbol + "_EVENTPIPE_Context"
+            Clrallevents.write('EXTERN_C __declspec(selectany) EVENTPIPE_TRACE_CONTEXT ' + eventpipeProviderCtxName + ' = { W("' + providerName + '"), 0, false, 0 };')
```

Comment:
Thanks @noahfalk! I refactored the script changes so that the changes are contained to a single file (clretwallmain.h). I created two temporary branches in my fork and committed the generated header for both Windows/Linux. You can easily see the diff by looking at: https://github.com/sywhang/coreclr/compare/temp...temp-2